### PR TITLE
Head cam config

### PR DESCRIPTION
**Pull request by benmaidel**

### CODE REVIEW

**Comment by benmaidel** [comment]
depends on https://github.com/ipa320/cob_calibration_data/pull/129

File: cob_hardware_config/cob4-7/config/image_flip_params_head_cam.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # rotation mode (0=fixed angle as defined  by rotation_angle, 1=automatic upright rotation against gravity using reference coordinate frame with upwards directed z-axis, 2=as 1 but image rotation only in 90deg steps -> faster)
 # int
-rotation_mode: 1
+rotation_mode: 0
```
Comment (floweisshardt) [inline cob_hardware_config/cob4-7/config/image_flip_params_head_cam.yaml]: we should keep mode 1 here as we'd like to flip the image as soon as the torso starts bending.

**Review by floweisshardt — CHANGES_REQUESTED** [review]

**Comment by benmaidel** [comment]
@ipa-fmw removed image_flip changes

File: cob_bringup/drivers/usb_camera_node.launch [hunk 2]
```diff
@@ -6,11 +6,11 @@
...
 	<node name="$(arg camera_name)" pkg="usb_cam" type="usb_cam_node" output="screen">
 		<param name="video_device" value="/dev/video0"/>
-		<param name="image_width" value="2048"/>
-		<param name="image_height" value="1536"/>
+		<param name="image_width" value="1280"/>
+		<param name="image_height" value="960"/>
 		<param name="pixel_format" value="yuyv"/>
 		<param name="camera_frame_id" value="$(arg camera_name)_link"/>
-		<param name="framerate" value="3.5"/>   <!-- at 2048x1536 3.6hz is maximum on hw -->
+		<param name="framerate" value="10"/>   <!-- at 2048x1536 3.6hz is maximum on hw -->
```
Comment (fmessmer) [inline cob_bringup/drivers/usb_camera_node.launch]: maybe it makes sense to move these parameters into a yaml file within `cob_hardware_config` as they are camera-specific and this is a general usb_camera_node launch file...

do we achieve this rate on the robot?
also the hz_monitor.yaml needs to be adjusted to the updated framerate!

**Review by fmessmer — CHANGES_REQUESTED** [review]

**Review by fmessmer — APPROVED** [review]
@ipa-bnm please merge!

**Comment by benmaidel** [comment]
Tested on HW cob4-7

**Pull request merged**